masterlist.yaml

```
@@ -6867,3 +6867,27 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'Another Pine Forest (Mod|Trees Only|Winter)\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/54027/' ]
+    inc: [ 'Boston Natural Surroundings.esp' ]
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Another Pine Forest plugin' ]
+        condition: 'many("Another Pine Forest (Mod|Trees Only|Winter)\.esp")'
+    dirty:
+      # Main v0.3
```

You don't need to (& it doesn't make sense to) put cleaning data in a regex plugin when it only applies to a specific plugin. Also, version comments aren't necessary as those will be recorded in the commit message:

```suggestion
  - name: 'Another Pine Forest Mod.esp'
    dirty:
```